flow/project.py

```diff
@@ -3046,8 +3039,6 @@ def _add_placeholder_operation(job):
         if profiling_results:
             print("\n" + "\n".join(profiling_results), file=file)
 
-        return status_renderer
```

Comment:
since `print_status` is public in python api, I think it might be worth mentioning that we now don't return anything when this method is called